### PR TITLE
[EA Forum only] email users asking them to fill out the annual user survey

### PR DESCRIPTION
**Pull request by s-cheng**
This PR sets up an email that we will send to users asking/reminding them to fill out the user survey. See more details and copy in [this doc](https://docs.google.com/document/d/1AV-ahepwFrd42RyRM_gnyOvDgTk40OpR2BdzeSG0mhw/edit). Copy to be finalized after Toby has a chance to look at it.

Example email:
Screen Shot 2024-08-19 at 1 57 39 AM

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1208088670456507) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/lib/collections/users/schema.ts [hunk 1]
```diff
@@ -3051,7 +3051,7 @@ const schema: SchemaType<"Users"> = {
     ...schemaDefaultValue(false),
   },
   
-  // EA Forum emails the user a survey if they haven't read a post in 3 months
+  // EA Forum emails the user a survey if they haven't read a post in 4 months
```
Comment (s-cheng) [inline packages/lesswrong/lib/collections/users/schema.ts:3054]: Fixed this comment

File: packages/lesswrong/server/repos/UsersRepo.ts [hunk 2]
```diff
@@ -486,6 +486,59 @@ class UsersRepo extends AbstractRepo<"Users"> {
...
+   /**
+   * Returns a list of users to whom we want to send an email
+   * asking them to fill in the annual EA Forum user survey.
+   *
+   * This excludes deleted/deactivated users,
+   * flagged or purged or removed from queue users,
+   * users who unsubscribed from all site emails,
+   * users who were banned any time over the past 6 month period,
+   * users who haven't been to the site in the past 2 years,
+   * and users who have already been sent this email.
```
Comment (s-cheng) [inline packages/lesswrong/server/repos/UsersRepo.ts:499]: I'd be interested in feedback on these criteria (in addition to checking for bugs in the query below), if you have any

Comment (Will-Howard) [reply packages/lesswrong/server/repos/UsersRepo.ts:499]: I've checked for bugs and I'm reasonably confident there aren't any in the current query. I think those criteria are good. You could maybe add negative karma too?

**Review by Will-Howard — APPROVED** [review]
LGTM. I tried rendering the email and check the repo query and it all looked good. I didn't try the script to actually send the email

File: packages/lesswrong/server/repos/UsersRepo.ts (hunk 2 above)
Comment (s-cheng) [reply packages/lesswrong/server/repos/UsersRepo.ts:499]: Good idea, I'll add that!

**Pull request merged**